fabtests/benchmarks/benchmark_shared.c

```diff
@@ -267,28 +275,46 @@ int bandwidth(void)
 	return 0;
 }
 
-static int bw_rma_comp(enum ft_rma_opcodes rma_op)
+static int bw_rma_comp(enum ft_rma_opcodes rma_op, int valid_windows)
 {
 	int ret;
 
 	if (rma_op == FT_RMA_WRITEDATA) {
+		/* for write data, only the client sends,
+		   and only the server verifies. */
 		if (opts.dst_addr) {
 			ret = bw_tx_comp();
 		} else {
 			ret = rma_bw_rx_comp();
 		}
-	} else {
+	} else if (rma_op == FT_RMA_WRITE) {
+		ret = ft_get_tx_comp(tx_seq);
+
+		if (ft_check_opts(FT_OPT_VERIFY_DATA)) {
+			/* ensure remote has gotten their completions. */
+			ft_sync();
+			ret = ft_check_buf((char *) rx_buf + offset_rma_start,
+				   opts.transfer_size*valid_windows);
+			if (ret) return ret;
```

Comment:
Place return on its own line, though it doesn't look needed.

Reply:
Thank you, I will fix the formatting.

I don't follow how it's not needed.  I'm trying to ensure ft_check_buf didn't find any errors, and it returns -FI_EIO on failure.